app/model.py

```diff
@@ -1012,52 +1018,50 @@ def create_id():
         return unique_id.key().id()
 
 class UsageCounter(db.Model):
-    """Counters count the historical statistics for a given repository.
-    To see how this is used, check out admin_stastics.py.
-    Unlike the Counter class, UsageCounter object increment when
+    """Counters which count the historical statistics for each repository.
+    To see how this is used, check out admin_statistics.py.
+    Unlike the Counter class, UsageCounter object increments when
     a new record or a new note is created, which means, the UsageCounter
     will not decrement when a record/note is expired/deleted."""
 
     # repo stored as a seperate property so it can be indexed and queried.
     repo = db.StringProperty(required=True)
-    # person_counter increment when a new person record is created.
+    # Total number of person records created so far,
+    # including deleted/expired ones.
     person_counter = db.IntegerProperty(default=0, required=True)
-    # note_counter increment when a new note is added in a person's record.
+    # Total number of notes created so far, including deleted/expired ones.
     note_counter = db.IntegerProperty(default=0, required=True)
 
     @staticmethod
```

Comment:
It's probably better to use @classmethod here. I believe @classmethod is more useful than @staticmethod in general, and there's no benefit to mix these two.

I know that we already mix @classmethod and @staticmethod in our code base, but let's only use @classmethod for new code :)

Same for below.